python/lsst/ip/isr/isrFunctions.py

```diff
@@ -444,3 +445,46 @@ def overscanCorrection(ampMaskedImage, overscanImage, fitType='MEDIAN', order=1,
         raise pexExcept.Exception('%s : %s an invalid overscan type' % \
             ("overscanCorrection", fitType))
     ampImage -= offImage
+
+
+def attachTransmissionCurve(exposure, opticsTransmission=None, filterTransmission=None,
+                            sensorTransmission=None, atmosphereTransmission=None):
+    """Attach a TransmissionCurve to an Exposure, given separate curves for different components.
+
+    Parameters
+    ----------
+    exposure: `lsst.afw.image.Exposure`
```

Comment:
I believe we're supposed to have a space either side of the colon:

    exposure : `lsst.afw.image.Exposure`